postUser/PostController.js

```diff
@@ -0,0 +1,64 @@
+import User from "./user.js";
+
+class PostController {
+  async create(req, res) {
+    try {
+      const { userName, email, password, role } = req.body;
+      const post = await User.create({ userName, email, password, role });
+      res.json(post);
+    } catch (e) {
+      res.status(500).json(e);
+    }
+  }
+
+  async getAll(req, res) {
+    try {
+      const posts = await User.find();
+      return res.json(posts);
+    } catch (e) {
+      res.status(500).json(e);
+    }
+  }
+  async getOne(req, res) {
```

Comment:
not good name for this function.. is you are searching by id then your function should called smth like getById()